browser/azure-storage.blob.export.js

```diff
@@ -62,6 +66,7 @@ module.exports.Logger = azureCommon.Logger;
 module.exports.WebResource = azureCommon.WebResource;
 module.exports.Validate = azureCommon.validate;
 module.exports.date = azureCommon.date;
```

Comment:
Why module.exports.date is in lowercase?

Reply:
Keep align (non-breaking) with Node.js exporting which is history code. https://github.com/Azure/azure-storage-node/blob/692ac1edb2dd01cb9bfd486b2cf89d7aa7e1e76c/lib/azure-storage.js#L364